gulpfile.babel.js

```diff
@@ -80,6 +80,7 @@ gulp.task('dev:generate', () => {
   return gulp.src(['lib/app/css/*.css'])
     .pipe(styleguide.generate({
       title: 'SC5 Styleguide',
+      sideNav: true,
```

Comment:
Will this be the new default value, so if you update toNav changes to sideNav? If so, I would change default value to false.